client/signup/steps/design-picker/index.jsx

```diff
@@ -49,24 +72,35 @@ class DesignPickerStep extends Component {
 		let designs = undefined;
 
 		if ( this.props.showOnlyThemes ) {
-			// Only offering designs that are also available as themes. This means excluding
-			// designs where the `template` has a layout that's different from what the theme's
-			// default Headstart annotation provides.
-			designs = getAvailableDesigns().featured.filter(
-				( { features, template, theme } ) => theme === template && ! features.includes( 'anchorfm' )
-			);
+			// Should probably be pulled out into a utility to format theme response as design picker design
+			// taxonomies.theme_subject probably maps to category
+			designs = this.props.themes
+				.filter( ( { id } ) => ! EXCLUDED_THEMES.includes( id ) )
+				.map( ( { id, name } ) => ( {
+					categories: [],
+					features: [],
+					is_premium: false,
+					slug: id,
+					template: id,
+					theme: id,
+					title: name,
+					...( STATIC_PREVIEWS.includes( id ) && { preview: 'static' } ),
+				} ) );
 		}
 
 		return (
-			<DesignPicker
-				designs={ designs }
-				theme={ this.props.isReskinned ? 'light' : 'dark' }
-				locale={ this.props.locale } // props.locale obtained via `localize` HoC
-				onSelect={ this.pickDesign }
-				className={ classnames( {
-					'design-picker-step__is-large-thumbnails': this.props.largeThumbnails,
-				} ) }
-			/>
+			<div>
```

Comment:
Why do we need this?

Reply:
Sorry, I think this might have been remnants from where I was experimenting with the category sidebar. It can be removed from this PR